src/app/src/sass/05_base/_base.scss

```diff
@@ -25,7 +25,7 @@ body {
     bottom: 0;
     left: 0;
     width: 100%;
-    overflow: hidden;
+    overflow: none;
```

Comment:
`overflow: auto` looks good on my Chrome's ipad mode

Reply:
Thanks! Pushed a commit to implement this!